packages/app/src/components/nominationBanner/style.css

```diff
@@ -111,66 +111,68 @@
   overflow: inherit;
   background: #fff;
   box-sizing: border-box;
-  padding: 20px;
   box-shadow: 0 1px 5px rgba(0,0,0,0.7);
   border-radius: 4px;
-  width: 469px;
-  height: 250px;
-  display: flex;
+  width: 800px;
   flex-direction: column;
-  justify-content: space-evenly;
-  align-items: center;
-  align-content: flex-start;
   display: flex;
   pointer-events: visible;
 }
 
 .modal-text {
-  padding: 40px 40px;
-  font-weight: 300;
+  padding: 20px 20px;
+  font-weight: 250;
   letter-spacing: .01em;
+  align-items: center;
 }
 
 .modal-buttons {
   display: flex;
-  justify-content: center;
-  align-items: center;
   line-height: 20px;
-  padding-bottom: 19px;
+  align-items: flex-end;
+  justify-content: flex-end;
 }
 
-
 .modal-buttons > button {
   color: white;
   border: #999;
   font-weight: bold;
-  padding: 10px 40px 10px 40px;
   margin-right: 10px;
   height: 35px;
-  border-radius: 4rem;
+  border-radius: 1rem;
   font-size: 1rem;
   letter-spacing: .01em;
   line-height: 1rem;
+  text-transform: capitalize;
```

Comment:
Do we need this? The text doesn't look capitalized on the screenshot.

Reply:
text-transform: capitalize; I used this because the buttons(I assume by default)is written all in 
![Screen Shot 2021-08-09 at 7 40 39 PM](https://user-images.githubusercontent.com/63215085/128791347-ff1e1a6a-920c-4767-a00e-961b95b4b3f3.png)
 capital letters:

Reply:
Okay, thanks for clarifying. In the future, it'd be helpful to have the screenshot match the latest code otherwise it's hard to tell if the CSS is working correctly.

Reply:
Sorry I didn't update the screen shot. Here is an updated screen shot.I've check with product and they are ok with the design.
![Screen Shot 2021-08-11 at 8 12 09 PM](https://user-images.githubusercontent.com/63215085/129123190-0bff0834-5a89-4637-9f84-c6dc2bc964b0.png)

Reply:
Seems like the button text switched back to being non-capitalized in the last few screenshots but the text-transform rule is still there. Should the text be capitalized still?

Reply:
if I don't use text-transform: capitalize the text will be all in capital letter.There is any other way to have have the text just capitalized?

Reply:
Oh, my bad. I had it backwards. I thought `text-transform: capitalize` made it all uppercase. This looks good then.